README.md

```diff
@@ -57,49 +67,28 @@ try {
 }
 ```
 
-
 Calling `getAccount` with the existing account index, will retrieve the account stored on the device.
+
 ```java
 if (kinClient.hasAccount()) {
     account = kinClient.getAccount(0);
 }
-``` 
+```
 
 You can delete your account from the device using `deleteAccount`, 
 but beware! you will lose all your existing KIN if you do this.
+
 ```java
 kinClient.deleteAccount(int index);
-``` 
-
-### Onboarding
-A first step before an account can be used, is to create the account on Stellar blockchain and fund it with native Stellar
-asset for fees purposes, this funding should be done by the digital service servers.  
-When working against KIN test network, the Fee Faucet Service can be used for this purpose:
+```
 
-```java
-Request request = new Request.Builder()
-            .url("http://friendbot-kik.kininfrastructure.com/?addr=" + account.getPublicAddress())
-            .get()
-            .build();
-okHttpClient.newCall(request)
-    .enqueue(new Callback() {
-        @Override
-        public void onFailure(@NonNull Call call, @NonNull IOException e) {
-            e.printStackTrace();
-        }
+## Onboarding
 
-        @Override
-        public void onResponse(@NonNull Call call, @NonNull Response response) throws IOException {
-            if (response.code() == 200) {
-                //account was created successfully, continue to the second phase - account activation
-            }
-        }
-    });
-```
+Before an account can be used on the configured network, it must be funded with the native network asset,
```

Comment:
Mention the XLM/Lumen words here.

Reply:
not sure, it's a subject for a change, and it's not really a Lumen.